modules/management-console-agent/src/main/java/org/apache/ignite/agent/StompDestinationsUtils.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.agent;
+
+import java.util.UUID;
+
+/**
+ * Stomp destinations utils.
+ */
+public class StompDestinationsUtils {
+    /** Cluster topology destination. */
+    private static final String CLUSTER_PREFIX_DEST = "/app/agent/cluster";
+
+    /** Cluster topology destination. */
+    private static final String CLUSTER_TOPOLOGY_DEST = CLUSTER_PREFIX_DEST + "/topology/";
+
+    /** Cluster caches info destination. */
+    private static final String CLUSTER_CACHES_INFO_DEST = CLUSTER_PREFIX_DEST + "/caches/";
+
+    /** Cluster caches sql meta destination. */
+    private static final String CLUSTER_CACHES_SQL_META_DEST = CLUSTER_CACHES_INFO_DEST + "sql-meta/";
+
+    /** Cluster node configuration. */
+    private static final String CLUSTER_NODE_CONFIGURATION = CLUSTER_PREFIX_DEST + "/node-config/%s";
+
+    /** Cluster action response destination. */
+    private static final String CLUSTER_ACTION_RESPONSE_DEST = CLUSTER_PREFIX_DEST + "/action/";
+
+    /** Save span destination. */
+    private static final String SAVE_SPAN_DEST = "/app/agent/spans/%s/add";
+
+    /** Metrics destination. */
+    private static final String METRICS_DEST = "/app/agent/metrics/%s/add";
+
+    /** Events destination. */
+    private static final String EVENTS_DEST = CLUSTER_PREFIX_DEST + "/events/%s/add";
+
+    /**
+     * @param clusterId Cluster id.
+     * @return Cluster topology destination.
+     */
+    public static String buildClusterTopologyDest(UUID clusterId) {
+        return CLUSTER_TOPOLOGY_DEST + clusterId;
+    }
+
+    /**
+     * @param clusterId Cluster id.
+     * @return Cluster node configuration.
+     */
+    public static String buildClusterNodeConfigurationDest(UUID clusterId) {
+        return String.format(CLUSTER_NODE_CONFIGURATION, clusterId);
```

Comment:
Try to don't use `String.format`. This method generates more garbage. Use usual string concatenation.